featuretools/synthesis/dfs.py

```diff
@@ -48,11 +48,9 @@ def dfs(dataframes=None,
 
 
     Args:
-        dataframes (dict[str -> tuple(DataFrame, str, str,
-                                      dict[str -> str/Woodwork.LogicalType],
-                                      dict[str->str/set],
-                                      boolean)]): dictionary of DataFrames.
-            Entries take the format dataframe name -> (dataframe, index column, time_index, logical_types, semantic_tags, make_index)}.
+        dataframes (dict[str -> tuple(DataFrame, str, str, dict[str -> str/Woodwork.LogicalType], dict[str->str/set], boolean)]):
+            Dictionary of DataFrames. Entries take the format 
+            {dataframe name -> (dataframe, index column, time_index, logical_types, semantic_tags, make_index)}.
```

Comment:
This update looks fine, but if you look further down in the args list, the link to TransformPrimitive is being displayed differently for `groupby_trans_primitives` than it is for `trans_primitives`. Can we fix?

![image](https://user-images.githubusercontent.com/4307001/129211208-83f3ac62-734b-4399-8786-92b6073096db.png)